lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -360,7 +360,7 @@ def _prepare_environment(  # noqa: C901
                         location_name, log
                     )
                     for azure_cap in location_info.capabilities:
-                        if azure_cap.vm_size == node_runbook.vm_size:
+                        if node_runbook.vm_size.lower() in azure_cap.vm_size.lower():
```

Comment:
Couldn't this cause a conflict if node_runbook.vm_size is a prefix of azure_cap.vm_size ? For example node_runbook.vm_size = "Standard_A2" and azure_cap.vm_size = "Standard_A2m_v2"

Reply:
good catch, I will find a way to support it.